core/web/apiv2/bloom.py

```diff
@@ -0,0 +1,72 @@
+import requests
+from fastapi import APIRouter, HTTPException, Request
+from pydantic import BaseModel, ConfigDict
+
+from core.config.config import yeti_config
+
+
+class BloomSearchRequest(BaseModel):
+    model_config = ConfigDict(extra="forbid")
+    values: list[str]
+
+
+class BloomHit(BaseModel):
+    value: str
+    hits: list[str]
+
+
+# API endpoints
+router = APIRouter()
+
+BLOOMCHECK_ENDPOINT = yeti_config.get("bloom", "bloomcheck_endpoint")
+
+
+@router.post("/search")
+def search(httpreq: Request, request: BloomSearchRequest) -> list[BloomHit]:
+    """Checks the bloomcheck microservice for hits."""
+    try:
+        response = requests.post(
```

Comment:
This is a bit weird to me but if i understand it correctly, the Yeti API server makes an http call to the bloom service (which is separate), to get the status hit/no hit?